bitcoin/__init__.py

```diff
@@ -52,6 +54,7 @@ class RegTestParams(bitcoin.core.CoreRegTestParams):
     BASE58_PREFIXES = {'PUBKEY_ADDR':111,
                        'SCRIPT_ADDR':196,
                        'SECRET_KEY' :239}
+    BECH32_HRP = 'tb'
```

Comment:
[Should](https://github.com/satoshilabs/slips/blob/master/slip-0173.md) be `bcrt` for regtest